ansible/pmm2/post-build-actions.yml

```diff
@@ -5,6 +5,14 @@
   gather_facts: yes
 
   tasks:
+    - name: Detect distribution        | Create '/srv/pmm-distribution' file
+      copy:
+        content: "{{ ansible_virtualization_type }}"
+        dest: /srv/pmm-distribution
+
+    - name: Add env to supervisor      | Add environment variable of TELEMETRY_OS to supervisord config
```

Comment:
pmm-managed should read this file and drop this environment variable completely.

Reply:
Removed.